### PR TITLE
Add core community call 10

### PR DESCRIPTION
**Pull request by kore4lyf**
Here is the transcription for community call 10, which was held on  October 20, 2023 (Feature Gate Threshold Activation)

### CODE REVIEW

**Comment by kore4lyf** [comment]
@jacobcreech
I am sorry I didn't specify the document, I was working on earlier, before completing it. I was hoping I'd get a response after I applied for the RFP.

**Comment by kore4lyf** [comment]
@jacobcreech 

Here is what you should know about the document [call_10.md](./call-noted/call_10.md)

- The proposal that was used in the call, doesn't exist anymore both, in buffalojoec's and solana-foundation's repository copy, although I provided the link in the document anyway, but it won't lead to the actual document but to a not found GitHub page. 

- I have not been able to find Josh Siegel's profile. 

- There are some certain (very few) parts of the video where a word or two is mumbled by the speaker, making it really difficult to hear what was being said. I have replaced such parts with triple dots (...). 
One can easily find them by doing a quick search `...`


I will be waiting for your response, kindly going through the document and give me a feedback.

File: call-notes/call_10.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+**Joe C:** Cool sounds good thanks Jacob. Hey guys I'm Joe for those of you guys who haven't met me. Yeah, I just wanted to talk a little bit about the SIMD, if any of you guys haven't seen it, it's number 72 [[SIMD 0072](https://github.com/solana-foundation/solana-improvement-documents/pull/72)]. Should I just share my screen, What do you think Jacob or ?
+
+**Jacob Creech:** Go for it.  
+
+#### Overview
+
+Video |  [00:35](https://youtu.be/DOaQ_VBFyIw?t=35)
+-|-
+
+**Joe C:** Yeah, I am not going to go through every crazy nit bit detail but we can talk a little bit about the key point and stuff, whatever you guys kind of want to jam on but basically, this is the proposal [here](https://github.com/buffalojoec/solana-improvement-documents/simd-feature-gate-threshold-automation/proposals/0072-feature-gate-threshold-automation.md). If someone coud please provide the link, that'd be great.
```
Comment (jacobcreech) [inline call-notes/call_10.md:43]:
```suggestion
**Joe C:** Yeah, I am not going to go through every crazy nit bit detail but we can talk a little bit about the key point and stuff, whatever you guys kind of want to jam on but basically, this is the proposal [here](https://github.com/buffalojoec/solana-improvement-documents/simd-feature-gate-threshold-automation/proposals/0072-feature-gate-threshold-automation.md). If someone could please provide the link, that'd be great.

```

**Comment by jacobcreech** [comment]
@kore4lyf there's a decent amount of filler words still kept in the transcript. Ideally I'd like those removed. These are things such as "like" and "right".

Great example would be 

```
And the whole idea is to basically, create a process that is going to be like automated, for kind of doing what we already have laid out with like the manual process for activating a feature
```

would turn into 

```
And the whole idea is to create a process that is going to be automated for doing what we already have laid out with the manual process for activating a feature
```

File: call-notes/call_10.md (hunk 1 above)
Comment (kore4lyf) [reply call-notes/call_10.md:43]: Alright I'd go through and correct all spelling errors.

**Comment by kore4lyf** [comment]
I wasn't sure if I was doing the right thing by not including a word like "um", due to the instruction provided in the RFP. 
Which is:
> You listen to the full call and transcribe it word to word

The removal of filler words should also be included in the instruction to assist future transcribers, in transcribing the calls as instructed.

**Review by jacobcreech — APPROVED** [review]
This is great! Thank you for your contribution!

**Pull request merged**